storage/gcs/src/main/java/io/aiven/kafka/tieredstorage/storage/gcs/MetricCollector.java

```diff
@@ -46,23 +46,23 @@ class MetricCollector {
      * <p>That is, {@literal "/storage/v1/b/<bucket>/o/<object>"}.
      */
     private static final Pattern OBJECT_METADATA_PATH_PATTERN =
-        Pattern.compile("^/storage/v1/b/\\w+/o/\\w+/?$");
+        Pattern.compile("^/storage/v1/b/(.*)/o/(.*)/?$");
```

Comment:
I guess this would be more correct
```suggestion
        Pattern.compile("^/storage/v1/b/([^/]+)/o/([^/]+)/?$");
```